src/features/upload-package-artifact/upload-package-artifact.ts

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
+import { UploadOptions } from '@actions/artifact';
+import { uploadArtifact } from '../../services/artifact-client';
+import { packPackage } from '../../services/npm-client';
+import { toAlphaNumeric } from '../../utils/formatters/string';
+
+export const uploadPackageArtifact = async (
+  packagePath: string,
+  options?: UploadOptions
+): Promise<ArtifactMeta> => {
+  const packageJsonPath = path.resolve(packagePath, 'package.json');
+
+  const { name } = require(packageJsonPath);
```

Comment:
Why do you need to read `package.json`, can't you take a name from a `packagePath`?

Reply:
Not sure what this means. I needed package name e.g. "@telia/b2b-logged-in-service". How would you go about  taking a name from a packagePath?

Reply:
And your `packagePath` is `services/b2b-logged-in-service`